deeppavlov/skills/aiml_skill/test_aiml_skill.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import unittest
+import os
+
+from deeppavlov.agents.default_agent.default_agent import DefaultAgent
+from deeppavlov.agents.processors.highest_confidence_selector import HighestConfidenceSelector
+from deeppavlov.skills.aiml_skill.aiml_skill import AIMLSkill
+
+
+class TestAIMLSkill(unittest.TestCase):
```

Comment:
We use `pytest`. Tests can be found in `/tests/`

Reply:
fixed